ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -70,6 +70,78 @@ tests:
           cpu: "4"
           memory: 4Gi
     workflow: ipi-gcp
+- as: win10-pipeline-example-test
+  run_if_changed: (^automation/.*|^templates-pipelines/windows-bios-installer/.*|^templates-pipelines/windows-customize/.*|^modules/.*|^templates/.*)$
+  steps:
+    cluster_profile: gcp-virtualization
+    env:
+      COMPUTE_NODE_TYPE: n2-standard-4
```

Comment:
Was this needed to speed things up or to get virtualization support? Is it covered by `gcp-virtualization` profile now?

Reply:
Mainly to get virtualization suport